splipy/SplineObject.py

```diff
@@ -1147,6 +1147,49 @@ def split(self, knots, direction=0):
 
         return results
 
+    def make_periodic(self, continuity=None, direction=0):
+        """Make the spline object periodic in a given parametric direction.
+
+        :param continuity: The continuity along the boundary (default max).
+        :param direction: The direction to ensure continuity in.
+        """
+
+        direction = check_direction(direction, self.pardim)
+        basis = self.bases[direction]
+        if continuity is None:
+            continuity = basis.order - 2
+        if not 0 <= continuity <= basis.order - 2:
+            raise ValueError('Illegal continuity for basis of order {}: {}'.format(
+                continuity, order
+            ))
+        if basis.periodic >= 0:
+            raise ValueError('Basis is already periodic')
+
+        basis = basis.make_periodic(continuity)
+
+        # Merge control points
```

Comment:
I had to update, turns out I was modifying the original CP array instead of a copy.

You're the expert on NURBS so I'll defer to you, but I'll just note that `cf.circle()` creates a C0-continuous object, and after `.split().make_periodic(0)` I seem to be getting back the identical one.